Subject: [PATCH] cmake4 fixes
---
Index: src/api.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/api.c b/src/api.c
--- a/src/api.c	(revision 2c891fc7a770e8ba2fec34fc6b545c672beb37e6)
+++ b/src/api.c	(date 1757420083181)
@@ -1,5 +1,6 @@

 #include "yaml_private.h"
+#include "string.h"

 /*
  * Get the library version.
Index: CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision 2c891fc7a770e8ba2fec34fc6b545c672beb37e6)
+++ b/CMakeLists.txt	(date 1757418791567)
@@ -1,5 +1,5 @@
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 project (yaml C)
 
 set (YAML_VERSION_MAJOR 0)
